File: admin/src/api/interface/systemDictType.ts
// system_dict_type 字典类型
import { ReqPage } from "./index";
export namespace SystemDictType {
  export interface ReqSystemDictTypeList extends ReqPage {
    type?: string; // 字典类型
    status?: number; // 状态（0正常 1停用）
    name?: string; // 字典名称
  }
  export interface ResSystemDictTypeItem {
    id: number; // 字典主键
    name: string; // 字典名称
    type: string; // 字典类型
    status: number; // 状态（0正常 1停用）
    remark: string | undefined; // 备注
    creator: string | undefined; // 创建人
    createTime: string | undefined; // 创建时间
    updater: string | undefined; // 更新人
    updateTime: string | undefined; // 更新时间
  }
}

File: cloud/dao/system_dict_type.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// SystemDictType 字典类型 system_dict_type
type SystemDictType struct {
	Id         *int64        `db:"id,-" json:"id"`                //bigint 字典主键,PRI
	Name       *string       `db:"name" json:"name"`              //varchar 字典名称
	Type       *string       `db:"type" json:"type"`              //varchar 字典类型
	Status     *int32        `db:"status" json:"status"`          //tinyint 状态（0正常 1停用）
	Remark     null.String   `db:"remark" json:"remark"`          //varchar 备注
	Creator    null.String   `db:"creator" json:"creator"`        //varchar 创建人
	CreateTime null.DateTime `db:"create_time" json:"createTime"` //datetime 创建时间
	Updater    null.String   `db:"updater" json:"updater"`        //varchar 更新人
	UpdateTime null.DateTime `db:"update_time" json:"updateTime"` //datetime 更新时间
}
